File: 2020/01/day01_2.rb
report = []
file = File.new("input.txt", "r")
while (line = file.gets)
  report.push(line.to_i)
end
file.close

report.each do |i|
  report.each do |j|
    report.each do |k|
      if( i + j + k== 2020)
        puts i*j*k
        exit
      end
    end
  end
end

File: 2020/01/day01_2.py
report = []
file = open("input.txt", "r") 
for line in file:
  report.append(int(line.strip()))

for i in report:
  for j in report:
    for k in report:
      if i+j+k == 2020:   
        print(i*j*k)
        exit()